src/main/kotlin/org/stellar/walletsdk/Wallet.kt

```diff
@@ -15,4 +15,35 @@ class Wallet(private val horizonUrl: String, private val networkPassphrase: Stri
     val keypair: KeyPair = KeyPair.random()
     return AccountKeypair(keypair.accountId, String(keypair.secretSeed))
   }
+
+  // Fund (activate) account
+  // TODO: ??? do we need to include add trustline operation here?
+  fun fund(
```

Comment:
The function only returns the transaction. Shouldn't we submit the transaction to Horizon?

Reply:
There will be separate functions to sign and submit. That way developers will have better control of how many operations they want to add, how they want to sign, etc.